### PR TITLE
Added Rosetta to C-Chain integration tutorial

### PR DESCRIPTION
**Pull request by jpop32**

### CODE REVIEW

File: build/tutorials/platform/integrate-exchange-with-avalanche.md [hunk 1]
```diff
@@ -36,6 +38,12 @@ Interacting with the C-Chain is identical to interacting with [go-ethereum](http
 
 Please note that `personal_` namespace is turned off by default. To turn it on, you need to pass the appropriate command line switch to your node, like in the above config example.
 
+## Integration using Rosetta
+
+[Rosetta](https://www.rosetta-api.org/) is an open-source specification and set of tools that makes integrating with Ethereum-compatible blockchains easier by presenting the same set of APIs for every network.
```
Comment (patrick-ogrady) [inline build/tutorials/platform/integrate-exchange-with-avalanche.md:43]: Not just `ethereum-compatible`, with any blockchain

Comment (jpop32) [reply build/tutorials/platform/integrate-exchange-with-avalanche.md:43]: Oh? Wow. Ok, will correct.

Comment (patrick-ogrady) [reply build/tutorials/platform/integrate-exchange-with-avalanche.md]: here are some blurbs that could help explain:

```
Rosetta is an open-source specification and set of tools that makes integrating with blockchains simpler, faster, and more reliable.
```

```
The Rosetta API is made up of 2 core components, the Data API and the Construction API. Together, these APIs allow for anyone to read and write to blockchains in a standard format over a standard communication protocol. 
```

File: build/tutorials/platform/integrate-exchange-with-avalanche.md [hunk 2]
```diff
@@ -36,6 +38,12 @@ Interacting with the C-Chain is identical to interacting with [go-ethereum](http
 
 Please note that `personal_` namespace is turned off by default. To turn it on, you need to pass the appropriate command line switch to your node, like in the above config example.
 
+## Integration using Rosetta
+
+[Rosetta](https://www.rosetta-api.org/) is an open-source specification and set of tools that makes integrating with different blockchain networks easier by presenting the same set of APIs for every network.
```
Comment (patrick-ogrady) [inline build/tutorials/platform/integrate-exchange-with-avalanche.md:43]: Here are some more blurbs if useful:
```
Rosetta is an open-source specification and set of tools that makes integrating with blockchains simpler, faster, and more reliable.
```

```
The Rosetta API is made up of 2 core components, the Data API and the Construction API. Together, these APIs allow for anyone to read and write to blockchains in a standard format over a standard communication protocol. The specifications for these APIs can be found in the rosetta-specifications repository.
```

File: build/tutorials/platform/integrate-exchange-with-avalanche.md [hunk 3]
```diff
@@ -36,6 +38,12 @@ Interacting with the C-Chain is identical to interacting with [go-ethereum](http
...
+
+You can find the Rosetta server implementation for Avalanche C-Chain [here](https://github.com/ava-labs/avalanche-rosetta), all you need to do is install and run the server with proper configuration. You don't necessarily need to even run you own node, and can instead rely on our [Public API](https://docs.avax.network/build/tools/public-api) to provide network access.
```
Comment (patrick-ogrady) [inline build/tutorials/platform/integrate-exchange-with-avalanche.md:45]: Nit:

```
avalanche-rosetta is packaged as a single Dockerfile to make it easy to run and manage.
```

More color on this + online/offline mode here: https://www.rosetta-api.org/docs/node_deployment.html

Comment (patrick-ogrady) [reply build/tutorials/platform/integrate-exchange-with-avalanche.md:45]: I'd add that detailed setup instructions can be found on the repo

Comment (jpop32) [reply build/tutorials/platform/integrate-exchange-with-avalanche.md:45]: Added clarification.

File: build/tutorials/platform/integrate-exchange-with-avalanche.md [hunk 4]
```diff
@@ -36,6 +38,12 @@ Interacting with the C-Chain is identical to interacting with [go-ethereum](http
 
 Please note that `personal_` namespace is turned off by default. To turn it on, you need to pass the appropriate command line switch to your node, like in the above config example.
 
+## Integration using Rosetta
+
+[Rosetta](https://www.rosetta-api.org/) is an open-source specification and set of tools that makes integrating with different blockchain networks easier by presenting the same set of APIs for every network. The Rosetta API is made up of 2 core components, the Data API and the Construction API. Together, these APIs allow for anyone to read and write to blockchains in a standard format over a standard communication protocol. The specifications for these APIs can be found in the rosetta-specifications repository.
```
Comment (patrick-ogrady) [inline build/tutorials/platform/integrate-exchange-with-avalanche.md:43]: Let's fill in the links for Data API, Construction API, and rosetta-specifications (which can be found in this paragraph: https://www.rosetta-api.org/docs/welcome.html#start-building).

File: build/tutorials/platform/integrate-exchange-with-avalanche.md [hunk 5]
```diff
@@ -2,7 +2,9 @@
 
 ## Overview
 
-The objective of this document is to provide a brief overview of how to integrate with the EVM-Compatible Avalanche C-Chain. For teams that already support ETH, supporting the C-Chain is as straightforward as spinning up an Avalanche node \(which has the [same API](https://eth.wiki/json-rpc/API) as [go-ethereum](https://geth.ethereum.org/docs/rpc/server)\) and populating Avalanche’s ChainID \(43114\) when constructing transactions.
+The objective of this document is to provide a brief overview of how to integrate with the EVM-Compatible Avalanche C-Chain. For teams that already support ETH, supporting the C-Chain is as straightforward as spinning up an Avalanche node \(which has the [same API](https://eth.wiki/json-rpc/API) as [go-ethereum](https://geth.ethereum.org/docs/rpc/server)\) and populating Avalanche’s ChainID \(43114\) when constructing transactions. There is also a [Rosetta](https://www.rosetta-api.org/) implementation for Avalanche, which can be even easier to integrate.
```
Comment (patrick-ogrady) [inline build/tutorials/platform/integrate-exchange-with-avalanche.md:5]:
```
There is also a [Rosetta](https://www.rosetta-api.org/) implementation for Avalanche, which can be even easier to integrate.
```
>>>>
```
\n

Additionally, Ava Labs maintains an implementation of the [Rosetta API](https://www.rosetta-api.org/) for the C-Chain called [avalanche-rosetta](https://github.com/ava-labs/avalanche-rosetta). You can learn more about this standardized integration path on the attached Rosetta API website.
```

File: build/tutorials/platform/integrate-exchange-with-avalanche.md [hunk 6]
```diff
@@ -2,7 +2,9 @@
...
+
+## Direct integration with C-Chain
```
Comment (patrick-ogrady) [inline build/tutorials/platform/integrate-exchange-with-avalanche.md:7]: nit: `Integration using EVM Endpoints` (or something like that to stay consistent with other titles...I think `Direct Integration` is confusing)

**Review by cgcardona — APPROVED** [review]

**Pull request merged**